lib/ui/widgets/components/banner_connectivity_not_available.dart

```diff
@@ -0,0 +1,17 @@
+/// SPDX-License-Identifier: AGPL-3.0-or-later
+import 'package:aewallet/ui/util/main_appBar_icon_network_warning.dart';
+// Package imports:
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class BannerConnectivityNotAvailable extends ConsumerWidget {
```

Comment:
Good proposal. Just perhaps instead this new widget, add in MainAppBarIconNetworkWarning a new optional  param "Alignment alignment". By default, alignment could be center but in intro pages, you can pass ``Alignment.topRight``.

Reply:
And rename ``MainAppBarIconNetworkWarning``to ``IconNetworkWarning`` and move to your folder proposal ``lib/ui/widgets/components``